docs/snippets/all/tutorials/any_values.rs

```diff
@@ -7,35 +7,25 @@ use rerun::external::arrow;
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let rec = rerun::RecordingStreamBuilder::new("rerun_example_any_values").spawn()?;
 
-    let confidences = rerun::SerializedComponentBatch::new(
-        Arc::new(arrow::array::Float64Array::from(vec![1.2, 3.4, 5.6])),
-        rerun::ComponentDescriptor::new("confidence"),
-    );
-
-    let description = rerun::SerializedComponentBatch::new(
-        Arc::new(arrow::array::StringArray::from(vec!["Bla bla bla…"])),
-        rerun::ComponentDescriptor::new("description"),
-    );
-
-    // URIs will become clickable links
-    let homepage = rerun::SerializedComponentBatch::new(
-        Arc::new(arrow::array::StringArray::from(vec![
-            "https://www.rerun.io",
-        ])),
-        rerun::ComponentDescriptor::new("homepage"),
-    );
-
-    let repository = rerun::SerializedComponentBatch::new(
-        Arc::new(arrow::array::StringArray::from(vec![
-            "https://github.com/rerun-io/rerun",
-        ])),
-        rerun::ComponentDescriptor::new("repository"),
-    );
-
-    rec.log(
-        "any_values",
-        &[confidences, description, homepage, repository],
-    )?;
+    let any_values = rerun::AnyValues::default()
+        // Using Rerun's builtin components.
+        .with_component::<rerun::components::Scalar>("confidence", [1.2, 3.4, 5.6])
+        .with_component::<rerun::components::Text>("description", vec!["Bla bla bla…"])
+        // Using arbitrary Arrow data.
+        .with_field(
+            "homepage",
+            Arc::new(arrow::array::StringArray::from(vec![
+                "https://www.rerun.io",
+            ])),
+        )
+        .with_field(
+            "repository",
+            Arc::new(arrow::array::StringArray::from(vec![
+                "https://github.com/rerun-io/rerun",
+            ])),
```

Comment:
If we use `impl Into<ArrayRef>` as the argument type, maybe we can get rid of the `Arc::new` here?

Reply:
I tried this, but unfortunately could not get the compiler to comply.